docker/wait.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/bash
+#!/bin/bash 
 # TODO convert to using docker compose'depends_on' feature with health checks
 
-set -e
+set -ex
```

Comment:
I'm assuming this was for debugging. Is it still needed? If not, please revert.

Reply:
yes reverting them

Reply:
It looks like `set -e` was there before this PR (same with `docker/wait.sh`), but it was removed in https://github.com/dimagi/commcare-hq/pull/26913/commits/17a88fe6222e6720519150188ff7bea25cdb3aaa. Was that intentional? I think it should probably stay unless something changed in this PR that requires the removal.